examples/purchase-tester/src/main/java/com/revenuecat/purchasetester/MainApplication.kt

```diff
@@ -42,7 +42,7 @@ class MainApplication : Application(), UpdatedCustomerInfoListener {
         val message = "CustomerInfoListener received update at ${customerInfo.requestDate}"
         Toast.makeText(this,
             message,
-            Toast.LENGTH_LONG
+            Toast.LENGTH_SHORT
```

Comment:
we get these so often and i found that they were delaying other, more time-sensitive toasts.